OnlineFlightSearchAPITestCases/ControllerTests/SearchFlightsControllerTests.cs

```diff
@@ -19,16 +20,21 @@ public SearchFlightsControllerTests()
                 .UseStartup<Startup>());
         }
 
+        // Will Ideally return StatusCode 200 OK. Since No Data is present ValidationMessage is thrown hence the BadRequest
         [Theory]
         [InlineData("BUD", "LTN")]
-        public async Task SearchFlightDetails_ValidRequest_ReturnsHttpStatusNoContent204(string startLocation, string destination)
+        public async Task SearchFlightDetails_ValidRequest_ReturnsHttpStatusBadRequest400(string startLocation, string destination)
```

Comment:
can we add the data to repo before testing?

Reply:
Mocked the Dependency of Controller and tested the OK Result.